Incompatibility with dask 2024.11.2

Two tests suite currently fails with the latest version of `dask` (2024.11.2):
```
=========================== short test summary info ============================
FAILED eliot/tests/test_dask.py::DaskTests::test_persist_pandas - AttributeError: 'HighLevelGraph' object has no attribute 'clear'
FAILED eliot/tests/test_dask.py::AddLoggingTests::test_add_logging_to_full_graph - AttributeError: 'HighLevelGraph' object has no attribute 'clear'
========== 2 failed, 479 passed, 16 skipped, 5086 warnings in 50.74s ===========
```

--- a/docs/source/news.rst
+++ b/docs/source/news.rst
@@ -8,6 +8,10 @@ Enhancements:
 
 * Added official support for Python 3.13, though 1.15.0 works fine too.
 
+Changes:
+
+* Tests pass with latest Dask.
+
 1.15.0
 ^^^^^^
 

--- a/eliot/dask.py
+++ b/eliot/dask.py
@@ -6,6 +6,7 @@ from dask import compute, optimize, persist
 
 try:
     from dask.distributed import Future
+    from dask.highlevelgraph import HighLevelGraph
 except:
 
     class Future(object):
@@ -108,6 +109,9 @@ def _add_logging(dsk, ignore=None):
 
     @return: New Dask graph.
     """
+    if isinstance(dsk, HighLevelGraph):
+        dsk = dsk.to_dict()
+
     ctx = current_action()
     result = {}
 

